nearlib/account.js

```diff
@@ -151,5 +151,40 @@ class Account {
     async viewAccount (accountId) {
         return await this.nearClient.viewAccount(accountId);
     }
+
+    /**
+     * Returns full details for a given account.
+     * @param {string} accountId id of the account to look up 
+     */
+    async getAccountDetails(accountId) {
+        accountId; // make linter happy
+        // Hardcoded for now
+        const result = {
+            authorizedApps : [
+                {
+                    id: 'cryptomonsters',
+                    name: 'Monsters',
+                    balance: 10,
+                },
+                {
+                    id: 'pay_mo',
+                    name: 'Paymo',
+                    balance: 20 
+                }
+            ],
+            activity : [
+                {
+                    id: '24359uy1dfsgwr',
+                    activity_type: 1, // to be converted to human-readible value on front end
```

Comment:
let's use `camelCase` everywhere in JS instead of `snake_case`, unless there are compatibility reasons to do otherwise

Reply:
done